tests/solidity/suites/exception/test/revert.js

```diff
@@ -0,0 +1,31 @@
+const TestRevert = artifacts.require("TestRevert")
+const truffleAssert = require('truffle-assertions');
+
+async function expectRevert(promise) {
+  try {
+    await promise;
+  } catch (error) {
+    if (error.message.indexOf('revert') === -1) {
+      expect('revert').to.equal(error.message, 'Wrong kind of exception received');
+    }
+    return;
+  }
+  expect.fail('Expected an exception but none was received');
+}
+
+contract('TestRevert', (accounts) => {
+  let revert
+
+  beforeEach(async () => {
+    revert = await TestRevert.new()
+  })
+  it('should revert', async () => {
+      await revert.try_set(10)
+      no = await revert.query()
```

Comment:
This will just throw and end the execution. You should use the `expectRevert` function which is defined above

Reply:
the test case actually success on ganache, only fail on ethermint.

Reply:
`await revert.set(10)` this can success on ganache?